plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiConfig.java

```diff
@@ -42,8 +42,8 @@ public void testDefaults()
                 .setStandardSplitWeightSize(DataSize.of(128, MEGABYTE))
                 .setMinimumAssignedSplitWeight(0.05)
                 .setMaxSplitsPerSecond(Integer.MAX_VALUE)
-                .setMaxOutstandingSplits(1000)
-                .setSplitLoaderParallelism(4)
+                .setMaxOutstandingSplits(10000)
+                .setSplitLoaderParallelism(10)
```

Comment:
While this is OK short-term, let's understand the bottleneck of split generation.

Also, increasing the max outstanding splits potentially increases the memory required for storing the splits on the coordinator.  Let's be vigilant on these.